src/components/tools/text-tool.tsx

```diff
@@ -302,6 +302,12 @@ export default class TextToolComponent extends BaseComponent<IToolTileProps, ISt
     return this.props.model.content as TextContentModelType;
   }
 
+  private handleOnBlur = () => {
+    this.setState({ editing: false });
+    const change = {args:[{text: this.getContent().text}]};
```

Comment:
One question I have is whether I should define a type for this and add it to this list here:  https://github.com/concord-consortium/collaborative-learning/blob/master/src/lib/logger.ts#L133 I'm not required to since this fits the shape of an existing, tile specific one (DrawingToolLoggingEvent).

Reply:
`DrawingToolLogEvent` looks pretty generic. I would consider moving it into `logger.ts` and renaming it something like `SimpleToolLogEvent` and then using it in both the text and drawing tiles.

Reply:
That's fine, but it's small enough that I also think you could just do it in this PR.

Reply:
Oh. Yeah. When I actually went to do it I realized it is small. Added to this PR.